cfme/cloud/tenant.py

```diff
@@ -313,16 +312,10 @@ def create(self, name, provider, wait=True):
         else:
             page.form.cancel_button.click()
 
-        all_view = self.create_view(TenantAllView)
-        wait_for(lambda: all_view.is_displayed, num_sec=120, delay=3,
-                 fail_func=all_view.flush_widget_cache, handle_exception=True)
+        all_view = self.create_view(TenantAllView, wait="20s")
```

Comment:
@digitronik Is this related to the `TODO` chage?

Reply:
No but I would like to put in same PRs. While working on one PR; If i found old implementation and no need to run test then I introduce in same PR to avoid multiple PRs.